packages/venia-ui/lib/components/SearchPage/searchPage.js

```diff
@@ -48,70 +51,52 @@ const SearchPage = props => {
                 </div>
             );
         }
-    }
 
-    let content;
-    if (data.products.items.length === 0) {
-        content = (
-            <div className={classes.noResult}>
-                <FormattedMessage
-                    id={'searchPage.noResultImportant'}
-                    defaultMessage={'No results found!'}
-                />
-            </div>
-        );
-    } else {
-        content = (
-            <Fragment>
-                <section className={classes.gallery}>
-                    <Gallery items={data.products.items} />
-                </section>
-                <section className={classes.pagination}>
-                    <Pagination pageControl={pageControl} />
-                </section>
-            </Fragment>
-        );
-    }
-
-    const totalCount = data.products.total_count || 0;
+        if (data.products.items.length === 0) {
```

Comment:
Previously it was not possible to hit this conditional if data is falsy, but now it technically can. I don't know how likely it is but should we guard against it with a `return null`?